app/models/Users.php

```diff
@@ -95,7 +104,7 @@ public function beforeValidationOnCreate()
         } else {
             $this->active = 'Y';
```

Comment:
Should we create a class constant for the Y & N?

Reply:
Thought about that, maybe will change here - #101